scripts/validate_string_concatenation.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python
+"""
+GH #30454
+
+Check where there is a string that needs to be concatenated.
+
+This is necessary after black formating,
+where for example black transforms this:
+
+>>> foo = (
+...         "bar "
+...         "baz"
+...     )
+
+into this:
+
+>>> foo = ("bar " "baz")
+
+Black is not considering this as an
+issue (see issue https://github.com/psf/black/issues/1051),
+so we are checking it here.
+"""
+
+import argparse
+import os
+import sys
+import token
+import tokenize
+from typing import Dict, Generator, List, Tuple
+
+FILE_EXTENSIONS_TO_CHECK: Tuple[str, str, str, str] = (
+    ".py",
+    ".pyx",
+    ".pyx.ini",
+    ".pxd",
+)
+
+
+def main(*, source_path: str = ".", output_format: str) -> bool:
```

Comment:
Why do we need the `*`?

Reply:
We don't, it's some leftover from when I built the script, just to organize my mind.

Removed from code.